styles/styles.css

```
@@ -32,10 +47,40 @@ li {
 }
 
 .navigation {
+    display: flex;
+    width: 1200px;
+    margin: 0 auto;
     background-color: #FFFFFF;
     color: #000000;
 }
 
+.navigation-list{
+    display: flex;
+    margin-left: 70px;
+    margin-right: 70px;
+    margin-top: 20px;
+    margin-bottom: 20px;
+    padding: 0;
+    width: 339px;
```

```suggestion
    max-width: 500px;
```
500 я поставила просто так.
Посмотри по макету на сколько могут расширяться правое и левое меню при увеличении количества пунктов.